1_OCaml/TP/TP1.ipynb

```diff
@@ -20,6 +20,78 @@
     "Puis utilisez ces fonctions sur des arguments de votre choix."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        (f -. 1. /. f) /. (f +. 1. /. f)
</pre><p>Mettre un in à la fin de ta fonction (pour avoir un séparateur entre ta définition de fonction et son utilisation) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>